packages/app/src/specs/CreateSpecModal.vue

```diff
@@ -77,6 +77,11 @@ fragment CreateSpecModal on Query {
     id
     fileExtensionToUse
     defaultSpecFileName
+    codeGenGlobs {
+      id
+      component
+    }
```

Comment:
With your refactor, you can remove `codeGenGlobs` from the fragment.

Reply:
I think that still needs to be in the query because I'm passing `gql.currentProject` via props to `ComponentGeneratorStepOne` and then using `gql.codeGenGlobs` there

Reply:
Ah yes, I missed that the fragment in ComponentGeneratorStepOne was on CurrentProject. 

Should we import the `ComponentGeneratorStepOne_codeGenGlob` fragment here and use it in this fragment so that the prop expectations don't get out of sync? Or do the types keep these aligned for us?

Reply:
I think this should be the pattern. It's confusing when a component has GraphQL properties that aren't actually used there - much better to import the fragment and spread it.

I wonder if we can lint for "no unused GraphQL properties".

Reply:
Okay I updated this to use the `ComponentGeneratorStepOne_codeGenGlob` fragment from `ComponentGeneratorStepOne`. Looks less confusing now